packages/async-flow/src/replay-membrane.js

```diff
@@ -28,12 +29,29 @@ const { fromEntries, defineProperties, assign } = Object;
  * @param {(vowish: Promise | Vow) => void} arg.watchWake
  * @param {(problem: Error) => never} arg.panic
  */
-export const makeReplayMembrane = ({
+export const makeReplayMembrane = arg => {
+  const noDunderArg = /** @type {typeof arg} */ (
+    Object.fromEntries(Object.entries(arg).filter(([k]) => !k.startsWith('__')))
+  );
+  return makeReplayMembraneForTesting(noDunderArg);
+};
+
+/**
+ * @param {object} arg
+ * @param {LogStore} arg.log
+ * @param {Bijection} arg.bijection
+ * @param {VowTools} arg.vowTools
+ * @param {(vowish: Promise | Vow) => void} arg.watchWake
+ * @param {(problem: Error) => never} arg.panic
+ * @param {boolean} [arg.__eventualSendForTesting] CAVEAT: Only for async-flow tests
```

Comment:
I wish there was an easier way to "extend" options while being able to use JSDoc to document the additional option (the only one I'm aware of is multiple `typedef` which is not clean)